src/main/java/frc/robot/subsystems/leds/LEDSubsystem.java

```diff
@@ -11,12 +11,17 @@ public class LEDSubsystem extends SubsystemBase {
 
     private final Timer blinkTimer;
     private static final double BLINK_DURATION_SECONDS = 0.5;
+    private final double APRIL_BLINK_DURATION = .2;
     private static final Color BLINK_COLOR = new Color(0, 0, 0);
     private boolean blinking = false;
+    private boolean continuous = false;
```

Comment:
is this used anywhere? i'm struggling to read this new feature, it took me a bit to understand what continuous even meant, and i'm not sure how users are expected to interface with the methods in this class and also LEDStrip.java. can you please try documenting what methods are meant to be for, along with maybe your logic?

Reply:
this is only used during the manual control mode, it is a toggle on the right joystick button